packages/console/src/components/SubmitFormChangesActionBar/index.tsx

```diff
@@ -0,0 +1,29 @@
+import Button from '../Button';
+import * as styles from './index.module.scss';
+
+type Props = {
+  isOpen: boolean;
+  isSubmitting?: boolean;
+  onDiscard: () => void;
+};
+
+const SubmitFormChangesActionBar = ({ isOpen, isSubmitting, onDiscard }: Props) => {
+  if (!isOpen) {
+    return null;
+  }
+
+  return (
+    <div className={styles.container}>
+      <Button size="medium" title="general.discard" onClick={onDiscard} />
```

Comment:
nit: can we call discard when isSubmitting?

Reply:
Good call; We should guard this case.